src/appengine/handlers/cron/triage.py

```diff
@@ -168,7 +209,7 @@ def is_crash_important(testcase):
           data_types.FILE_UNREPRODUCIBLE_TESTCASE_MIN_CRASH_THRESHOLD)
 
 
-def is_similar_bug_open_or_recently_closed(testcase, issue_tracker):
+def _is_similar_bug_open_or_recently_closed(testcase, issue_tracker):
```

Comment:
nit: name doesn't make it obvious it could update an existing issue.

how about something like:  "_check_and_update_similar_bugs()" ?

Reply:
Done!